### PR TITLE
Add prop for selected tab style

### PR DESCRIPTION
**Pull request by andoru**
I understand tab implementation may be changing soon but I've added a prop so the selected tab can have differing style.

### CODE REVIEW

File: src/TabBar.js [hunk 1]
```diff
@@ -57,7 +57,10 @@ class TabBar extends Component {
                 renderIcon={() => <Icon {...this.props} {...el} />}
                 renderSelectedIcon={() => <Icon {...this.props} {...el} selected />}
                 onPress={() => this.onSelect(el)}
-                tabStyle={el.tabStyle}
+                tabStyle={[
+                  el.tabStyle,
+                  isSelected ? el.tabSelectedStyle : null
```
Comment (charpeni) [inline src/TabBar.js]: Eslint Error - Missing trailing comma. (comma-dangle)

Comment (andoru) [inline src/TabBar.js]: Fixed.

**Comment by charpeni** [comment]
I would let @joenoon review this since he's working on tab.

**Comment by rturk** [comment]
@andoru I haven't tested it, so please correct me if I'm wrong this is a breaking change?

**Comment by joenoon** [comment]
@andoru I dont think this still would apply to master, so closing for now.  If you want to reopen a new one against master please do.

**Pull request closed without merging**